src/Chapter06.Tests/CompilerAssert2.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+using System.Collections.Immutable;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Shared;
+
+public static class CompilerAssert2
```

Comment:
I assume at some point we should come back in and clean this up as I did it more as just an example code.